examples/js/Ocean.js

```diff
@@ -253,16 +250,14 @@ THREE.Ocean.prototype.generateSeedPhaseTexture = function() {
 
 THREE.Ocean.prototype.renderInitialSpectrum = function () {
 
-	this.scene.overrideMaterial = this.materialInitialSpectrum;
 	this.materialInitialSpectrum.uniforms.u_wind.value.set( this.windX, this.windY );
 	this.materialInitialSpectrum.uniforms.u_size.value = this.size;
-	this.renderer.render( this.scene, this.oceanCamera, this.initialSpectrumFramebuffer, true );
+	this.THREE.WebGLRendererUtils.renderOverride( renderer, this.materialInitialSpectrum, this.scene, this.oceanCamera, this.initialSpectrumFramebuffer, true );
```

Comment:
Fixed that.  So you are okay with the refactor our of WebGLRenderer?  Sweet.

I've renamed WebGLRendererUtils to be EffectRenderer and moved it into postprocessing -- this makes sense to me.  It is just an easy way to do post effects anyhow without resorting to Pass-based system.